WcaOnRails/db/migrate/20160701034833_reference_table_updates.rb

```diff
@@ -0,0 +1,67 @@
+class ReferenceTableUpdates < ActiveRecord::Migration
+  def up
+    change_table :Rounds do |t|
+      t.boolean :final, null: false
+    end
+
+    ActiveRecord::Base.connection.execute("update Rounds set final = 1 where id in ('c', 'f');")
```

Comment:
As the Rounds table has appropriate model now why wouldn't we use Rails DSL
```ruby
Round.where(id: ['c', 'w']).update_all(final: 1)
```
Same below.